features/cellular/framework/targets/QUECTEL/BC95/QUECTEL_BC95.cpp

```diff
@@ -95,6 +95,16 @@ nsapi_error_t QUECTEL_BC95::init()
     return _at->unlock_return_error();
 }
 
+nsapi_error_t QUECTEL_BC95::set_baud_rate_impl(int baud_rate)
+{
+    return _at->at_cmd_discard("+NATSPEED", "=", "%d%d%d%d%d%d%d", baud_rate, 30, 0, 3, 1, 0, 1);
```

Comment:
Manual has a comment that sync mode does not support 3 when <baud rate> is 2400, 4800 and 57600.